src/_includes/layouts/home.liquid

```diff
@@ -1,86 +1,76 @@
 ---
 layout: layouts/base.liquid
 ---
-<section>
 
-  <div class="bg-parallax">
-    <div>
-      <img src="/assets/img/bg-triangle-gray.png" />
-    </div>
+<div class="relative overflow-clip">
+  <div class="absolute z-[-1]">
+    <img src="/assets/img/bg-triangle-gray.png" alt="Gray triangle"/>
   </div>
-
-  <div class="">
-    <div class="">
-      <div class="">
-        <h1 class="text-blue">{{ homeHeroRight.heroTitle }}</h1>
-        <h3 class="text-gray">{{ homeHeroRight.heroSubTitle }}</h3>
+  <div class="z-1 py-5 mx-8 desktop:mx-[8rem] bg-illustration bg-no-repeat h-[85vh]" style="background-image: url('/assets/img/illustration-gov.png'); background-size: contain;">
+    <div class="grid items-center desktop:grid-cols-2">
+      <div class="py-5 desktop:col-start-2 desktop:row-start-1 desktop:mt-[8rem] bg-white bg-opacity-90">
+        <h1 class="text-blue mb-8 leading-0 text-5xl">{{ homeHeroRight.heroTitle }}</h1>
+        <h3 class="text-grayLight text-2xl mb-6 leading-tight">{{ homeHeroRight.heroSubTitle }}</h3>
```

Comment:
Curious if we're going to go through an accessibility review here. We should have an `h2` next after the `h1`.

Reply:
If it is possible to use Tailwind for those changes, that would be preferred since we're already using Tailwind extensively.